src/main/scala/io/viash/schemas/CollectedSchemas.scala

```diff
@@ -26,18 +26,23 @@ import io.viash.functionality._
 import io.viash.runners._
 import io.viash.engines._
 import io.viash.platforms._
-import io.viash.functionality.arguments._
 import io.circe.Json
 import monocle.function.Cons
 import io.viash.config.Config
-import io.viash.config.Info
-import io.viash.functionality.resources._
+import io.viash.config.BuildInfo
 import io.viash.packageConfig.PackageConfig
 import io.viash.helpers._
 import scala.collection.immutable.ListMap
-import io.viash.functionality.dependencies._
 import io.viash.runners.nextflow.{NextflowConfig, NextflowAuto, NextflowDirectives}
 import io.viash.engines.requirements._
+import io.viash.config.arguments.{Argument, StringArgument, BooleanArgument, BooleanTrueArgument, IntegerArgument, LongArgument, BooleanFalseArgument, DoubleArgument, FileArgument}
+import io.viash.config.dependencies.{GithubRepository, GithubRepositoryWithName, LocalRepository, RepositoryWithName, ViashhubRepository, Dependency, LocalRepositoryWithName, GitRepositoryWithName, GitRepository, Repository, ViashhubRepositoryWithName}
+import io.viash.config.resources.{JavaScriptScript, CSharpScript, Resource, Executable, ScalaScript, NextflowScript, PlainFile, BashScript, PythonScript, RScript}
```

Comment:
Why not just

```suggestion
import io.viash.config.arguments._
import io.viash.config.dependencies._
import io.viash.config.resources._
```

?